src/js/core/methods/CardanoSignTransaction.js

```diff
@@ -41,13 +42,14 @@ export default class CardanoSignTransaction extends AbstractMethod {
         validateParams(payload, [
             { name: 'inputs', type: 'array', obligatory: true },
             { name: 'outputs', type: 'array', obligatory: true },
-            { name: 'fee', type: 'string', obligatory: true },
-            { name: 'ttl', type: 'string', obligatory: true },
+            { name: 'fee', type: 'amount', obligatory: true },
+            { name: 'ttl', type: 'string'},
             { name: 'certificates', type: 'array', allowEmpty: true },
             { name: 'withdrawals', type: 'array', allowEmpty: true },
             { name: 'metadata', type: 'string' },
             { name: 'protocolMagic', type: 'number', obligatory: true },
             { name: 'networkId', type: 'number', obligatory: true },
+            { name: 'validityIntervalStart', type: 'string'},
```

Comment:
I would move it after metadata since it is part of tx body by CDDL, unlike protocolMagic etc.